### ISSUE TITLE
allow more control on animation

### ISSUE THREAD

**Issue by christophe-g** [material-components/material-web#3712]
In view of https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html, would it be possible to provide more/better control over MD components animation? 

For instance, would it be possible to express all animation duration as a css variable so as to contol it over a `@media (prefers-reduced-motion: reduce)`  global statement (inspired for example by https://www.silvestar.codes/articles/using-css-variables-for-reduced-motion-on-a-global-scale/ or https://web.dev/learn/accessibility/motion/) ? 

We are using this strategy with success at https:accessiblesurveys.com on our own components. For the time being, we override all MD components for which we need to control animation behaviors. 

It would be simpler to provide this by default for the release of the next MD spec (MD3).

**Comment by asyncLiz** [comment]
You're absolutely correct! While we don't have them know, animation tokens are planned by design after layout tokens, so these will likely be customizable with custom properties.
